ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/components/video/VideoComponentState.kt

```diff
@@ -235,12 +243,10 @@ internal class VideoComponentState(
 
     @JvmSynthetic
     fun update(
-        windowSize: WindowWidthSizeClass? = null,
```

Comment:
Hmm I'm not sure this is right... in the other components states, the `update` method includes the `screenCondition`, so we can update it, but not for the VideoComponentState. Is there a reason for that difference? This also means that we will keep using the `initialScreenCondition` for the `PresentedPartial`  even if the screen changes, so not sure if that's right... Maybe we're handling that in the view layer?